integr8ly/cloud-resource-operator
# INTLY-8992 bump operator sdk v0.19.0

## Overview
bump operator sdk v0.19.0 
Jira: https://issues.redhat.com/browse/INTLY-8992

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make run`
you may see
```
[Deprecation Notice] Operator SDK has a new CLI and project layout that is aligned with Kubebuilder.
```
Kubebuilder is coming in v1.0.0 of the operator-sdk can be ignored for now.

- Run `make cluster/seed/managed/blobstorage`
- Run `make cluster/seed/managed/postgres`
- Run `make cluster/seed/managed/redis`

- Ensure that all resources are created on aws
